compiler/tests/testSuite.bal

```diff
@@ -57,15 +68,41 @@ function testCompileEU(string path, string kind) returns file:Error|io:Error? {
     }
 }
 
-function compileErrorLineNumber(CompileError err) returns int? {
-    if err is io:Error {
-        return ();
+function moduleDir(string filePath) returns string|file:Error? {
+    string subModPath = filePath.substring(0, filePath.length()-4) + ".modules";
+    if check file:test(subModPath, file:IS_DIR) {
+        return check file:normalizePath(subModPath, file:CLEAN);
+    }
+}
+
+function expectedErrorLocation(CompileError err, string path) returns [int, string]|file:Error|io:Error? {
```

Comment:
[string, int] is more natural than [int, string] here

Reply:
fixed